scripts/mockapi.js

```diff
@@ -81,9 +81,44 @@ const fetchDynamicStockIndexData = () => [
   },
 ];
 
+function getTrendingNews() {
+  const newsData = [
+    {
+      imgUrl: 'https://www.icicidirect.com/images/HG Infra-202403190956262997726.png',
```

Comment:
should we use sample published images placed in our repository? these links might not work after a while

Reply:
it's mock data..we any way need to clean that.